### PR TITLE
[User Accounts] Enforce site selection

### PR DESCRIPTION
**Pull request by davidblader**
https://redmine.cbrain.mcgill.ca/issues/12673

### CODE REVIEW

File: modules/user_accounts/test/user_accountsTest.php [hunk 1]
```diff
@@ -212,6 +215,8 @@ function testAddNewUser()
         $this->assertEquals($field->getAttribute('value'), 'last');
         $field = $this->safeFindElement(WebDriverBy::Name('Email'));
         $this->assertEquals($field->getAttribute('value'), 'email@example.com');
+        $field = $this->safeFindElement(WebDriverBy::Name('CenterIDs[]'));
```
Comment (kongtiaowang) [inline modules/user_accounts/test/user_accountsTest.php]: remove 218 -219

File: modules/user_accounts/test/user_accountsTest.php [hunk 2]
```diff
@@ -204,6 +204,9 @@ function testAddNewUser()
         $field->clear();
         $field->sendKeys('email@example.com');
         $this->safeClick(WebDriverBy::Name('SendEmail'));
+        $field = $this->safeFindElement(WebDriverBy::Name('CenterIDs[]'));
```
Comment (kongtiaowang) [inline modules/user_accounts/test/user_accountsTest.php]: 207-209
        $SitesElement =  $this->safeFindElement(
                                  WebDriverBy::Name('CenterIDs[]')
                               );
        $SitesOption = new WebDriverSelect($SitesElement);
        $SitesOption->selectByValue("1");

Comment (davidblader) [inline modules/user_accounts/test/user_accountsTest.php]: thank you!

**Comment by driusan** [comment]
It looks fine to me, but I don't see any original code review done in the history, just testing..

@MounaSafiHarab since it's multisite-related would you mind having a quick look and then I'll merge it?

**Pull request merged**